spinnman/transceiver.py

```diff
@@ -2776,8 +2845,14 @@ def scamp_connection_selector(self):
     @property
     def bmp_connection(self):
         """
+        This method is currently deprecated and untested as there is no
+        known use. Same functionaility provided by ybug and bmpc.
+        Retained in case needed for hardware debugging.
+
```

Comment:
Not sure I even understand what this returns!

Reply:
marked likely to be removed.